src/NLog/Targets/FileTarget.cs

```diff
@@ -1953,8 +1833,9 @@ private string FindSuitableFilename(string fileName, int numberToStartWith)
                 }
                 return targetFileName;
             }
-        }
 
+            private readonly Queue<string> fileQueue;
```

Comment:
is there a reason to have the fields below the methods? We prefer the following order: fields, ctor, properties, methods